src/components/Attachment.svelte

```diff
@@ -49,6 +51,14 @@
   function previewFiles(e) {
     error = false;
     function readAndPreview(file) {
+      errorMessage = "";
+
+      if (file.size > 5000000) {
```

Comment:
Maybe we can declare a `const` variable somewhere so that this can be easily changed in the future?